var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -104,6 +104,11 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
         description="Enable breakpad crash dump library",
         when="@1.10:1.11",
     )
+    # py-torch has strict dependencies on old protobuf/py-protobuf versions that
+    # don't build with some of the compilers (e.g. intel) or cause problems with
```

Comment:
If there are known build issues for protobuf, can you add a conflict to that package? This will allow the concretizer to automatically enable this variant when needed